x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -7701,7 +7701,6 @@
     "xpack.enterpriseSearch.schema.errorsTable.heading.error": "错误",
     "xpack.enterpriseSearch.schema.errorsTable.heading.id": "ID",
     "xpack.enterpriseSearch.schema.errorsTable.link.view": "查看",
-    "xpack.enterpriseSearch.schema.existingField.status.recentlyAdded": "最近添加",
```

Comment:
FYI: I'll re-add this i18n translations back in when App Search manages it's own "Recently added" string/ID